### ISSUE TITLE
pip creates pkg_resources.VersionConflict for Python 3.2 when tox explicitly installed

### ISSUE THREAD

**Issue by jayvdb** [ryanhiebert/tox-travis#26]
`pip install tox-travis` works well with the new virtualenv version management feature, however `pip install tox tox-travis` fails, which is more typically going to be used like `pip install tox==2.1.1 tox-travis`.

This is only a limitation on the new auto Python 3.2 support (inc. PyPy3 2.4), and doesnt affect the PyPy3 5.2 new support.  As such, this doesnt appear to be a regression, and doesnt affect the main feature of tox-travis 0.5 which is simplifying adoption of the new PyPy3. Unless someone reports that tox-travis 0.5 is breaking their build, this bug is not a high priority.

The reason is the dependency version resolution system in pip isnt very good. Upgrading to the newest pip and setuptools doesnt fix the problem.  Here is a log showing pip deciding to installing virtualenv 15.0.2 in order to meet requirement `virtualenv>=1.11.2 (from tox)`, effectively ignoring tox-travis the requirement from tox-travis.
https://travis-ci.org/jayvdb/pyflakes/jobs/148206747#L155

Anyone using `pip install tox tox-travis`, needs to change this to `pip install tox-travis` to use the new Python 3.2 support.

If someone really needs a custom version of tox, and Python 3.2 support is also needed, the workaround is to use the old approach for Python 3.2, by installing the "virtualenv<14" explicitly.

I'm pretty sure I can see a solution, but wont be working on it in the next ~5hrs unless someone notices a regression.

**Comment by jayvdb** [comment]
Actually, there is a regression, in that `Python: pypy3` and `pip install tox tox-travis` now fails.
Fixing..

**Comment by jayvdb** [comment]
So, the simple approach in #27 doesnt work to fix the regression :/

So the workaround is to explicitly install tox, then tox-travis, like

```bash
$ pip install tox==2.1.1
...
Successfully installed pluggy-0.3.1 tox-2.1.1 virtualenv-15.0.2
$ pip install tox-travis
...
Successfully installed tox-travis-0.5 virtualenv-13.1.2
```

**Comment by ryanhiebert** [comment]
I'm having trouble reproducing this on my local machine. I haven't tried getting a PR to test it on Travis, though.

I'm running pip version 7.1.2 in a virtualenv with PyPy 2.4.0 providing Python 3.2.5, and am seeing no issues running `pip install tox tox-travis`.

**Comment by ryanhiebert** [comment]
Hmm. The worst that I've been able to reproduce so far is a warning that support for Python 3.0 - 3.2 is being dropped. That's a message from pip, but so far I haven't seen the VersionConflict error.

**Comment by ryanhiebert** [comment]
@jayvdb: Is this still worth pursuing?

**Comment by rpkilby** [comment]
@ryanhiebert: my two cents are that python 3.2 is EOL, so... `¯\_(ツ)_/¯`

**Comment by ryanhiebert** [comment]
Works for me. If anyone comes along and really wants it, I'll merge fixes for Python 3.2, since Travis still supports it. I wonder if they'll ever stop... Perhaps we need to call 3.2 dead for our project too? I know virtualenv doesn't support it anymore.

**Comment by agriffis** [comment]
I just ran into this problem with `pypy3`. The workaround from @jayvdb worked for me, though I didn't use a specific tox version:

```yaml
language: python
python:
  - "2.7"
  - "3.3"
  - "3.4"
  - "3.5"
  - "3.6"
  - "pypy"
  - "pypy3"
install:
  # install tox-travis explicitly after tox to avoid
  # pkg_resources.VersionConflict
  - pip install tox
  - pip install tox-travis
```

Here's the failing case:
https://travis-ci.org/scampersand/bienvenue/jobs/196462565

and with the workaround:
https://travis-ci.org/scampersand/bienvenue/jobs/196463584
